src/com/dirj/DirJ.java

```diff
@@ -3,38 +3,73 @@
 import java.io.File;
 import java.text.NumberFormat;
 import java.text.SimpleDateFormat;
-import java.util.Date;
-import java.util.Locale;
+import java.util.*;
 
 public class DirJ {
     private static final String DIR = "  <DIR>";
+    private static boolean onlyFiles = false;
+    private static boolean lowerCase = false;
+    private static boolean helpOption = false;
+
     public static void main(String[] args) {
+        for (String arg : args) {
+            switch (arg) {
+                case "/B":
+                    onlyFiles = true;
+                    break;
+                case "/L":
+                    lowerCase = true;
+                    break;
+                case "/?":
+                    helpOption = true;
```

Comment:
In fact you can print help message and stop processing right here. No need to use field for this option.